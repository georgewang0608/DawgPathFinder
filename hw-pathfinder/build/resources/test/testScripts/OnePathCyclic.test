# Find a path with Dijkstra's algorithm when
# there exist one path between the node in an acyclic graph

CreateGraph simpleGraph

AddNode simpleGraph A

AddEdge simpleGraph A A 1

FindPath simpleGraph A A