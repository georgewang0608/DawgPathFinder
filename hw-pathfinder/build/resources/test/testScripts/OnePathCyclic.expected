# Find a path with Dijkstra's algorithm when
# there exist one path between the node in an acyclic graph

created graph simpleGraph

added node A to simpleGraph

added edge 1.000 from A to A in simpleGraph

path from A to A:
total cost: 0.000

